cndocs/text-style-props.md

```
@@ -1,53 +1,69 @@
 ---
 id: text-style-props
-title: Text 样式属性
+title: Text Style Props
 ---
 
 ### 示例
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)